### DISCUSSION TITLE
Black screen in Mesh remote agent, but can move mouse in the remote desktop

### DISCUSSION THREAD

**Discussion by Dasno7** [Ylianst/MeshCentral#5541] [Issues]
I have setup MeshCentral and I successfully added two PCs. However, when I open a mesh remote agent desktop connection I get a black screen. Whenever I hoover over it is seems that I can move the cursor in the remote and all my inputs (keyboard, mouse clicks) work. The weird thing is that I can only move my cursor within a small square in the top right of my remote desktop, when I change the zoom or other display settings none of this changes. It doesn't matter whether I remote from one PC to the other.

To summarize: I'm able to establish a stable connection but I get to see no image and I can only hoover over the top left part of my remote desktop screen.

**Comment by si458** [comment]
are you using cloudflare DNS proxy, or cloudflare tunnels? #5302
sadly cloudflare are changing stuff left right and central and now control side of things have stopped working?

**Reply by Dasno7** [reply]
Cloudflare DNS proxy. Could explain some of the issues I’m getting…

**Reply by si458** [reply]
yeh way to fix at moment, is turn off dns proxy, then use lets encrypt,
but this will only work if your meshcentral server is open through firewall on 80+443
